dandischema/tests/test_metadata.py

```diff
@@ -53,7 +53,7 @@ def test_pydantic_validation(schema_dir):
 
 
 def test_json_schemakey_validation():
-    with pytest.raises(ValidationError) as exc:
+    with pytest.raises(JsonschemaValidationError) as exc:
```

Comment:
would you be so kind to also change then the line below (can't offer suggestion out of diff scope) to be
```
assert "'schemaKey' is a required property" in str(exc.value.errors)
```
instead of `.args[0]`? or if you like - I can push that change